File: Ejercicios Propuestos 1/Ejercicio 17/Ejercicio 17.jl
FactorExtra = 1.5

print("Ingrese la cantidad total de empleados que tiene la empresa: ")
NumEmpleados = parse(Int, readline())

print("Ingrese el pago por hora ordinaria: ")
ptas_hora = parse(Float64, readline())

for contador in 1:NumEmpleados
    println("Empleado número ", contador)
    print("Ingrese las horas trabajadas: ")
    horas_trabajadas = parse(Float64, readline())
    
    if horas_trabajadas <= 40
        salario = horas_trabajadas * ptas_hora
    else
        horas_extras = horas_trabajadas - 40
        salario = (40 * ptas_hora) + (horas_extras * ptas_hora * FactorExtra)
    end
    
    println("El salario del empleado ", contador, " es: ", salario)
    println("------------------------------------------------")
end

File: Ejercicios Propuestos 1/Ejercicio 17/Ejercicio17.java
import java.util.Scanner;

public class Ejercicio17 {
    
    static final int NumEmpleados = 50;
    static final double FactorExtra = 1.5;
    
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("¿Cuántos empleados tiene la empresa? ");
        int cantidad = scanner.nextInt();
        
        System.out.print("¿Cuánto paga por hora ordinaria? ");
        double ptas_hora = scanner.nextDouble();
        
        for (int i = 1; i <= cantidad; i++) {
            System.out.print("Horas trabajadas del empleado " + i + ": ");
            double horas_trabajadas = scanner.nextDouble();
            
            double salario;
            
            if (horas_trabajadas <= 40) {
                salario = horas_trabajadas * ptas_hora;
            } else {
                double horas_extras = horas_trabajadas - 40;
                salario = (40 * ptas_hora) + (horas_extras * ptas_hora * FactorExtra);
            }
            
            System.out.println("Empleado " + i + " - Salario: " + salario);
        }
    }
}